.gitignore

```diff
@@ -11,6 +11,6 @@ bench/example/
 bench-hist/
 bench-temp/
 .shake/
-ghcide
+ghcide*
```

Comment:
Why? Seems like it might catch a bit too much. I'd definitely just write ghcide-bench here as a second entry.

Reply:
there's ghcide-preprocessor as well. But I'll add them both